### ISSUE TITLE
Write new 'Configure Nodes for Networking' section

### ISSUE THREAD

**Issue by degville** [CanonicalLtd/maas-docs#33]
I've removed the following examples from the old networking documentation (now installconfig-network):
1. Creating a Bond Interface
2. Creating a VLAN Interface
3. Deleting an Interface

These were removed because they used the 'node-interfaces' API call, which has been deprecated (LP: #1568207). These examples need to be either re-written for the latest API call, or replicated and replaced with GUI equivalents.

**Comment by pmatulis** [comment]
The installconfig-network2.md page was removed today (#127) and this is what was salvaged. I'm not certain whether it is really all that useful for users. A lot of it is broken in the API and the `interfaces` command is instead something that would be of more interest to users, in addition to the 'dry_run' and 'verbose' boolean operators:

`maas admin machines allocate dry_run=True verbose=True interfaces=internal:subnet=172.16.99.0/24`

Salvaged material:

```
## Placement constraints

When you allocate a node through the API/CLI, or search for nodes in the web
UI, you can specify two kinds of constraints for the node's network placement:

1. `networks` specifies that the node you want must be connected to *all* of
   the given networks.
1. `not_networks` says that a node must *not* be connected to *any* of the
   given networks.

Constraints can identify a network in any of several ways. You may combine
these freely. Each is a way of referring to a specific network, just expressed
in different ways.

- "`network-name`": The name of a network as it was defined in the MAAS. The
  example is for the case where you have defined a network with the name,
`network-name`.
- "`ip:10.122.1.0`": An IP address in the network. This can be the network's
  base address, or its broadcast address, or any other IP address that falls
within the network. So `ip:10.122.1.0` identifies the same network as, for
example, `ip:10.122.1.99`.
- "`vlan:13`": a VLAN tag. This can only be used for VLANs, so the tag must
  be nonzero. The tag is a number between 1 to 4094 inclusive.
- "`vlan:0x0d`": a VLAN tag in hexadecimal notation. The valid range is from
  `0x1` to `0xffe` inclusive. The notation is case-insensitive and leading
  zeroes are ignored. So, `vlan:0x0d`, `vlan:0Xd`, `vlan:0xD`, and `vlan:0X0d`
  are all equivalent to `vlan:13`.
```

**Comment by pmatulis** [comment]
I've moved this to 'low priority' as, after having worked on these docs a great deal, in comparison to other things, this is just not important. I think we should get to it eventually in the web UI. The CLI equivalent is of even less import.

**Comment by degville** [comment]
fixed with #723
